src/xdp/trn_transit_xdp.c

```diff
@@ -454,6 +457,26 @@ static __inline int trn_process_inner_ip(struct transit_packet *pkt)
 
 	__be64 tunnel_id = trn_vni_to_tunnel_id(pkt->geneve->vni);
 
+	/*
+	* If target ip is towards a virtual subnet, send it up to user space
+	*/
+	struct network_key_t nkey;
+	struct network_t *net;
+	nkey.prefixlen = 96;
+
+	// "__u32 nip[3]" has 12 bytes altogether
+	// tunnel_id is bit-wise 64 bit and occupies the first 8 bytes
+	// tip (target inner ip) occupies the rest 4 bytes
```

Comment:
line 467 to 469 can be removed. I added these to remind myself how data is store. originally mizar does not have this.

Reply:
Removed